build.gradle

```diff
@@ -1,30 +1,37 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
+apply plugin: 'com.huawei.ohos.app'
 
+ohos {
+    compileSdkVersion 5
+    defaultConfig {
+        compatibleSdkVersion 5
+    }
+}
+	
 buildscript {
-    
     repositories {
-        google()
-        jcenter()
-        maven { url "https://jitpack.io" }
+        maven {
+            url 'https://repo.huaweicloud.com/repository/maven/'
+        }
+        maven {
+            url 'https://developer.huawei.com/repo/'
+        }
+		jcenter()
```

Comment:
Is jcenter() repo required in build.gradle?

Reply:
Removed in commit 5693b42057797e7071ebb39d5c531d4eef3453bd.